bridgestyle/arcgis/togeostyler.py

```diff
@@ -258,40 +258,52 @@ def processSymbolReference(symbolref, options):
         for layer in symbol["symbolLayers"][
             ::-1
         ]:  # drawing order for geostyler is inverse of rule order
-            symbolizer = processSymbolLayer(layer, symbol["type"], options)
-            if symbolizer is not None:
-                if layer["type"] in [
-                    "CIMVectorMarker",
-                    "CIMPictureFill",
-                    "CIMCharacterMarker",
-                ]:
-                    if symbol["type"] == "CIMLineSymbol":
-                        symbolizer = {
-                            "kind": "Line",
-                            "opacity": 1.0,
-                            "perpendicularOffset": 0.0,
-                            "graphicStroke": [symbolizer],
-                            "graphicStrokeInterval": symbolizer["size"] * 2,  # TODO
-                            "graphicStrokeOffset": 0.0,
-                            "Z": 0,
-                        }
-                    elif symbol["type"] == "CIMPolygonSymbol":
-                        maxX = symbolizer.get("maxX", 0)
-                        maxY = symbolizer.get("maxY", 0)
-                        stepX = layer.get("markerPlacement", {}).get("stepX", 0)
-                        stepY = layer.get("markerPlacement", {}).get("stepY", 0)
-                        marginX = stepX - maxX or symbolizer["size"] * 2
-                        marginY = stepY - maxY or symbolizer["size"] * 2
-                        symbolizer = {
-                            "kind": "Fill",
-                            "opacity": 1.0,
-                            "perpendicularOffset": 0.0,
-                            "graphicFill": [symbolizer],
-                            "graphicFillMarginX": marginX,
-                            "graphicFillMarginY": marginY,
-                            "Z": 0,
-                        }
-                symbolizers.append(symbolizer)
+            if layer["enable"]:
```

Comment:
I would write:

```suggestion
            if not layer["enable"]:
                return symbolizers
```
I've already seen that we are not using this "coding style". But here, early return would be really better IMO

Reply:
I understand it would be more readable, I'll do the change